Bug: Non-exhaustive pattern match in rsky-firehose binary — Ok(None) not covered

**Package: rsky-firehose v0.2.1
  File: rsky-firehose/src/main.rs:98
  Error code: E0004**

### Description

  `cargo check -p rsky-firehose` fails to compile the rsky-firehose binary due to a non-exhaustive match on the return value of rsky_firehose::firehose::read.

  The function signature is:

  `pub fn read(data: &[u8]) -> Result<Option<(Header, SubscribeRepos)>>`

  It legitimately returns Ok(None) in several cases:
  - Decoding errors for any message type (#commit, #handle, #tombstone, #account, #identity)
  - #sync messages (intentionally skipped per AT Protocol Sync v1.1)
  - #info messages (intentionally skipped)
  - Unknown/future message types

  However, the match in main.rs:98 only covers Ok(Some(...)) and Err(...), omitting the Ok(None) arm:
```
  match rsky_firehose::firehose::read(&message) {
      Ok(Some((_header, body))) => { /* ... */ }
      Err(e) => eprintln!("...", e),
      // Ok(None) not handled — compile error
  }
```

  ## Error Output
```
  error[E0004]: non-exhaustive patterns: `Ok(None)` not covered
    --> rsky-firehose/src/main.rs:98:11
     |
  98 |     match rsky_firehose::firehose::read(&message) {
     |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ pattern `Ok(None)` not covered
```
 ## Expected Fix

  Add an Ok(None) arm to the match to explicitly handle the "message was intentionally skipped" case:

  Ok(None) => (), // message was skipped (info, sync, unknown, or decode error)

  ## Impact

  The rsky-firehose binary cannot be compiled at all.

## Environment
MacOS 15.7.3
Rust version: 1.86

--- a/rsky-firehose/src/main.rs
+++ b/rsky-firehose/src/main.rs
@@ -268,6 +268,7 @@ async fn process(message: Vec<u8>, client: &reqwest::Client) {
                 };
             }
         }
+        Ok(None) => (),
         Err(error) => eprintln!(
             "@LOG: Error unwrapping message and header: {}",
             error.to_string()

